src/components/templates/PartyMap/components/ScheduleItem/ScheduleItem.scss

```diff
@@ -9,30 +9,48 @@
     background: $dark;
   }
 
-  .time-section {
+  &__time-section {
     border-right: 1px solid $concrete;
     min-width: 110px;
     margin-right: 10px;
   }
 
-  .event-description {
+  &__event-description {
+    display: block;
     color: $concrete;
     white-space: pre-wrap;
+
+    p {
+      display: inline;
+    }
```

Comment:
Why was this added here? We don't tend to write styles that target HTML elements like this.

Reply:
This was the only way I was able to find to present an addition blank line occuring after each event description in the event modal, while also allowing line breaks. If you have any hints for better ways to present line breaks like that, happy to implement differently.

Reply:
I thought I already replied to this.. but can't find it now. Was line breaks and similar not one of the original reasons we implemented markdown rendering? Why can't that be used here?

Reply:
Though a further question here is why do we need to allow line breaks in the first place?